pyhole/core/logger.py

```diff
@@ -30,22 +70,21 @@ def setup_logger(name):
 
     debug = utils.debug_enabled()
 
-    log_dir = utils.get_directory("logs")
     log_level = logging.DEBUG if debug else logging.INFO
```

Comment:
Unfortunately this will require more thought since the `debug_enabled()` call will cause stuff to fail if the config file doesn't exist.